evalml/tests/automl_tests/test_time_series_split.py

```diff
@@ -21,8 +21,18 @@ def test_time_series_split_init():
         _ = list(ts_split.split(X=pd.DataFrame(), y=pd.Series([])))
 
 
-def test_time_series_split_n_splits_too_big():
-    splitter = TimeSeriesSplit(gap=7, n_splits=4, max_delay=3, date_index=None)
+@pytest.mark.parametrize(
+    "gap,max_delay,forecast_horizon,n_splits",
+    [[7, 3, 1, 4], [0, 3, 2, 3], [1, 1, 1, 4]],
+)
+def test_time_series_split_n_splits_too_big(gap, max_delay, forecast_horizon, n_splits):
+    splitter = TimeSeriesSplit(
+        gap=gap,
+        max_delay=max_delay,
+        forecast_horizon=forecast_horizon,
+        n_splits=n_splits,
+        date_index="date",
+    )
     X = pd.DataFrame({"features": range(15)})
     # Each split would have 15 // 5 = 3 data points. However, this is smaller than the number of data_points required
```

Comment:
I think this comment needs to be updated since not every split is now using 3 data points.

Reply:
I'm just going to delete it. Don't think it's adding much value since it's obvious the values are incompatible given the expected behavior of the test.